Failed to build with Go 1.16

<!---
This issue tracker is mainly for bugs and feature requests. Before asking a
question, search online and try to investigate on your own.
-->

#### What versions are you running?

Latest [commit](https://github.com/chromedp/chromedp/tree/60742071de0f116803c6af2bb8c22337f562a250).

#### What did you do? Include clear steps.

[Install go1.16](https://go.dev/doc/manage-install) and run the command:

```go
go1.16 build ./...
```

#### What did you expect to see?

The build is successful.

#### What did you see instead?

An error:
```
conn.go:9:2: //go:build comment without // +build comment
```

#### Additional Info
Looks like it's because of [updating `ws` dependency](https://github.com/chromedp/chromedp/commit/60742071de0f116803c6af2bb8c22337f562a250),

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/chromedp/cdproto v0.0.0-20230220211738-2b1ec77315c9
-	github.com/gobwas/ws v1.2.0
+	github.com/gobwas/ws v1.2.1
 	github.com/ledongthuc/pdf v0.0.0-20220302134840-0c2507a12d80
 	github.com/mailru/easyjson v0.7.7
 	github.com/orisano/pixelmatch v0.0.0-20220722002657-fb0b55479cde

